### ISSUE TITLE
Discuss convention for all dynamic values in a11y strings be surrounded in <em> tags

### ISSUE THREAD

**Issue by zepumph** [phetsims/a11y-research#60]
From a skype discussion with @jessegreenberg as well as https://github.com/phetsims/chipper/issues/610. It would be nice to solidify a convention for `<em>` now, while only a few sims have descriptions in them. In OL it seemed to work very nicely to have dynamic properties be surrounded by emphasis. Then in the a11y view they were highlighted nicely. BASE doesn't have any `<em>` tags though. It would be good to make sure that if we continue down a path where `<em>` tags get green/underlining in the a11y view because they are dynamic, then we are consistent.

@terracoda what do you think?

**Comment by terracoda** [comment]
@zepumph, I like this idea. One thing to keep in mind is that some screen readers actually read out emphasized text in a different voice. I have noticed that VO reads content in `em` tags in a higher pitch, for example. For Ohm's Law this is ideal because there is a natural fit for emphasis. It may not work nicely for all dynamic strings, especially sentences with multiple dynamic strings.

Would it be possible to have a couple of options based on the context? For example, `<em>` or `<span class="dynamic"></span>`. If we went the named class route, that would have built-in flexibility as it could be placed on any element, block-level or in-line, that contains a dynamic description.

**Comment by zepumph** [comment]
I like that a lot. I wonder how we could go about adding that class easily to places that are inherently dynamic. My opinion would be to keep `class="dynamic"` out of the a11y string files as much as possible.

**Comment by jessegreenberg** [comment]
> My opinion would be to keep class="dynamic" out of the a11y string files as much as possible.

Agreed, especially since these strings will eventually be translatable.

**Comment by jessegreenberg** [comment]
I remember that we had a discussion about this a while back and couldn't find notes. I  remember @zepumph having a nice way to handle this, but can't remember exactly what was discussed. In the meantime, here is one potential way to do this:

- Add a query parameter to initialize-globals that indicates that sim is being run in a11y view, add this flag to query param list in sim-a11y-view.html.
- When in a11y view, iterate through {{SIM}}A11yStrings and wrap placeholder `{{}}` with `<i></i>`.
- When in a11y view, short circuit Accessibility.js functions `setAccessibleDescription` and `setAccessibleLabel` to `setAccessibleDescriptionAsHTML` and `setAccessibleLabelAsHTML`, so `<i>` tags are rendered in the a11y view.

PROS:
- All accessibility strings that are dynamic automatically get styled in a11y view.
- Requires no changes to simulation code.
- `<i>` tags do not have to be added to a11y strings in {{SIM}}A11yStrings.js/json files.

CONS:
- Maybe some strings that are not meant to  be dynamic for a11y purposes will get styled. For instance the "Grab {{balloonLabel}}" pattern in BASE is not really a dynamic string, but it is assembled with StringUtils.fillIn() for i18n.

**Comment by jessegreenberg** [comment]
I recently made https://github.com/phetsims/a11y-research/issues/67#issuecomment-358797815 in this issue but decided it is better in #67, so removing that comment from this thread.

**Comment by jessegreenberg** [comment]
In #67 we decided that all `<em>` tags that were used for styling should be `<i>` tags so that for consistency and so that  we can use `<em>` tags to provide emphasis when necessary. This issue can be closed.
